src/components.jsx/ArticleCard.jsx

```diff
@@ -0,0 +1,46 @@
+import {
+  Card,
+  CardBody,
+  CardFooter,
+  Heading,
+  Image,
+  Stack,
+  Button,
+  Text,
+  Divider,
+  ButtonGroup,
+} from "@chakra-ui/react";
+
+const ArticleCard = ({ article }) => {
+  return (
+    <Card maxW="lg">
+      <CardBody>
+        <Image
+          src={article.article_img_url}
+          alt="Green double couch with wooden legs"
+          borderRadius="lg"
+        />
+        <Stack mt="6" spacing="3">
+          <Heading size="md">{article.title}</Heading>
+          <Text>{article.body.slice(0, 50)}...</Text>
+          <div className="author-votes-container">
+            <Text color="teal" fontSize="md">
+              By {article.author}
+            </Text>
+            <Text>{article.votes} votes</Text>
+          </div>
+        </Stack>

```

Comment:
We also want to see when the articles were created in a human readable format as they should already be sorted by date when they come back from the api